test/overlays/overlay.yaml

```diff
@@ -11,7 +11,7 @@ actions:
       - wildcard-done
 - target: $.paths['/pets'].get
   update:
-    info:
+    x-info:
```

Comment:
Is this change because the info tag doesn't belong there in an operation?

Reply:
Yup, I didn't know if validity of the test case output was worth making the change for... Maybe we could swap it for an `externalDocs`object or something else which does apply there?